pagopa/io-app
# fix: [IA-870] Show the welcome screen only the first time for the user

## Short description
This PR makes sure that the welcome screen after the onboarding is shown only one time for the user.

## List of changes proposed in this pull request
- Call the complete onboarding saga only if `isProfileFirstOnBoarding` returns true;
- Remove the thank you screen from E2E tests.

## How to test
Logout -> Login -> The welcome screen should not be displayed.
Set `firstOnboarding` to `true` in the dev server to see the screen.
